glusterd2/events/liveness.go

```diff
@@ -2,12 +2,12 @@ package events
 
 import (
 	"strings"
-	"sync"
 
 	"github.com/gluster/glusterd2/glusterd2/store"
 
 	"github.com/coreos/etcd/clientv3"
 	log "github.com/sirupsen/logrus"
+	"fmt"
```

Comment:
I think this should be grouped with standard library packages (strings) instead of 3rd party packages